diff --git a/components/component_updater/component_updater_service_internal.h b/components/component_updater/component_updater_service_internal.h
index 341fbf78a96bb1b0b8e10bf9579dea9b930bec13..71fae6572ec91963f3325586fb27573f1ca65aef 100644
--- a/components/component_updater/component_updater_service_internal.h
+++ b/components/component_updater/component_updater_service_internal.h
@@ -69,7 +69,8 @@ class CrxUpdateService : public ComponentUpdateService,
   void Start();
   void Stop();
 
-  bool CheckForUpdates(UpdateScheduler::OnFinishedCallback on_finished);
+  virtual bool CheckForUpdates(UpdateScheduler::OnFinishedCallback on_finished);
+  friend class BraveCrxUpdateService;
 
   void OnDemandUpdateInternal(const std::string& id,
                               Priority priority,
